.travis-ci.sh

```diff
@@ -21,9 +21,10 @@ opam_install_test_deps () {
          result.1.4 \
          utop.2.4.2 \
          mdx.1.6.0
+    opam install \
```

Comment:
Do we need to install coq in a separate command? it can be installed as part of the other deps I believe

Reply:
I did this as to avoid the Travis timeout; that's a hairy subject as `travis_wait` won't work on the command, and Coq compilation in OCaml >= 4.08.1 got a bit slower.

This will hopefully fixed very soon tho as in 8.12 we hope to provide a `coq-core` package with only the ML parts, compilation time of that is much more reasonable.